src/Model/Contact.php

```diff
@@ -3252,6 +3258,7 @@ public static function addRelationship(array $importer, array $contact, array $d
 				'created'  => DateTimeFormat::utcNow(),
 				'url'      => $url,
 				'nurl'     => Strings::normaliseLink($url),
+				'baseurl'  => $pub_contact['baseurl'] ?? '',
```

Comment:
Same question about the related `gsid` field.

Reply:
We only need the `baseurl` to fetch the `gsid`.